frontend/index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
+    <base href="/"/>
     <meta charset="UTF-8" />
-    <link rel="icon" href="/favicon.ico" />
+    <link rel="icon" href="favicon.ico" />
```

Comment:
Not sure if this change was really necessary. Is it because it's more precise since the base tag only applies to relative URLs? Just wanted to note that `/favicon.ico` would've also worked (as far as I understand, only because the base in the vite config has been set to `./`).

Reply:
I still need to understand what Vite does here. I assumed it would only correct absolute paths when defining a base, leaving relative urls unchanged. I was wrong.
What I am really looking for is a way to use relative urls the way I intended them to be without Vite trying to be "intelligent".

Reply:
undone in d4d3ee6